governance/program/src/state/governance.rs

```diff
@@ -294,6 +314,10 @@ pub fn get_governance_data(
         // The assumption here is that council should have Veto vote enabled by default and equal to council_vote_threshold
         governance_data.config.council_veto_vote_threshold =
             governance_data.config.council_vote_threshold.clone();
+
+        // For legacy accounts default Council VoteTipping to the Community
+        governance_data.config.council_vote_tipping =
+            governance_data.config.community_vote_tipping.clone()
```

Comment:
I may be missing something in the upgrade path, but this is only triggered for legacy accounts where `governance_data.config.council_vote_threshold == VoteThreshold::YesVotePercentage(0)`.

Are there accounts where `council_vote_threshold` is set to something (ie, *not* `YesVotePercentage(0)`) that would use the default tipping rather than the community one?

Reply:
That condition triggers the upgrade path because it indicates we deserialised a legacy `V1` or `V2` account which didn't have settings for:
- `community_vote_threshold`
- `council_vote_threshold`
- `community_veto_vote_threshold`
- `council_veto_vote_threshold`
-  `community_vote_tipping`
-  `council_vote_tipping`

All these settings were introduced in `V3` and the code initialises them as follows:

- `community_vote_threshold`  replaces the legacy single `vote_threshold` field - the legacy field was renamed
- `council_vote_threshold` is set to legacy `vote_threshold` 
- `council_veto_vote_threshold ` is set to legacy `vote_threshold`
- `community_veto_vote_threshold` is set to `Disabled`
-  `community_vote_tipping` replaces the legacy singe `vote_tipping` field - the legacy field was renamed
- `council_vote_tipping` is set to the legacy `vote_tipping`

Note, it's not possible to set `council_vote_threshold` to `VoteThreshold::YesVotePercentage(0)` in `V3` and at the same time it's always that in the legacy versions and hence it can be used as the upgrade trigger.

Reply:
Ah ok, great, thanks for the explanation!